alarmee/src/commonMain/kotlin/com/tweener/alarmee/AlarmeeScheduler.kt

```diff
@@ -76,7 +90,18 @@ abstract class AlarmeeScheduler {
 
     internal abstract fun cancelAlarm(uuid: String)
 
-    private fun validateAlarmee(alarmee: Alarmee) {
+    abstract fun pushNotificationNow(alarmee: Alarmee, onSuccess: () -> Unit)
```

Comment:
This method should be `internal` so it cannot be called outside of the library.
The name of the method could be slightly improved to keep a consistent naming with the other internal methods:
```suggestion
    internal abstract fun pushAlarmee(alarmee: Alarmee, onSuccess: () -> Unit)
```
(I don't think the `Now` is necessary since it's redundant with "push".)

Reply:
That's a necessary change! Thanks for pointing it out!